appveyor.yml

```diff
@@ -0,0 +1,35 @@
+build: false
+skip_branch_with_pr: true
+platform: x64
+
+environment:
+  global:
+    PYTHONUNBUFFERED: "1"
+    INSTALL_EDM_VERSION: "3.0.1"
+
+  matrix:
+    - RUNTIME: '3.6'
+
+matrix:
+  fast_finish: true
+
+branches:
+  only:
+    - master
```

Comment:
If we remove this block then persumably we can run CI even if this PR does not target master.

Reply:
I removed it.  Should I revert back to have this PR target the branch for #4 now?